onnx_tf/handlers/backend/identity.py

```diff
@@ -20,3 +16,11 @@ def version_1(cls, node, **kwargs):
   @classmethod
   def version_13(cls, node, **kwargs):
     return [cls.make_tensor_from_onnx_node(node, **kwargs)]
+
+  @classmethod
+  def version_14(cls, node, **kwargs):
+    x = kwargs["tensor_dict"][node.inputs[0]]
+    if isinstance(x, (list, tuple)):
+      return [tf.identity_n(x)]
```

Comment:
Please note ONNX sequence allows variable shapes and tf.identity_n supports tensors with same shapes only.